cmake/CreateVirtualEnvironment.cmake

```diff
@@ -3,7 +3,7 @@
 CreateVirtualEnvironment
 ------------------------
 Function that creates a virtual Python environment with specific requirements.
-From https://schemingdeveloper.com/2020/07/02/how-to-create-a-new-python-virtual-environment-in-cmake/
+Inspired by https://schemingdeveloper.com/2020/07/02/how-to-create-a-new-python-virtual-environment-in-cmake/
```

Comment:
Note: I chopped this up a lot while trying to debug it, plus I needed to move the core logic to python for the fix (and ultimately that also helped diagnose the problem).

The original script is nice, but had a lot of features we didn't need that made it hard for me to follow and also fueled a lot of bogus theories.